.golangci.yml

```diff
@@ -27,6 +27,9 @@ run:
   # on Windows.
   skip-dirs:
     - .go
+    - pkg/inetdiag
```

Comment:
Why exclude these?

Reply:
A lot of the enum names are fully capitalized, since they are meant to mimic the Linux kernel enums. The linter doesn't like that and refuses to build with them as is. If there's a better way of doing this please let me know!